### DISCUSSION TITLE
Error when run  exe after use flet build windows

### DISCUSSION THREAD

**Discussion by abdokhalil07** [flet-dev/flet#4247] [Q&A]
### Question

i build windows app with "flet build windows" but exe file not work and give me error
, i try to 
![error](https://github.com/user-attachments/assets/d3567dd0-51ab-495d-a020-c49a7e824d64)
 build it with "auto-py-to-exe" and it work without any error

### Error message

```bash
Traceback (most recent call last):
  File "<string>", line 43, in <module>
  File "<frozen runpy>", line 229, in run_module
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\ABDELR~1\AppData\Local\Temp\serious_python_temp519fe14a\main.py", line 2, in <module>
ImportError: cannot import name 'NavigationBarDestination' from 'flet' (C:\Users\ABDELR~1\AppData\Local\Temp\apps-1.0.0-1\app\__pypackages__\flet\__init__.pyc)
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by LegendaryPistachio** [comment]
命令打包成功率低

**Comment by LegendaryPistachio** [comment]
The success rate of command packaging is low.

**Comment by ndonkoHenri** [comment]
Which python version do you have?

**Reply by abdokhalil07** [reply]
3.11.4

**Reply by ndonkoHenri** [reply]
And which Flet version. (run `flet --version`)

**Reply by abdokhalil07** [reply]
0.24.1
